File: Edinburgh College/JS/2. Conditional/activity2.js
firstNumber = Number(prompt("First number: "))
secondNumber = Number(prompt("Second number: "))
thirdNumber = Number(prompt("Third number: "))

if (secondNumber > firstNumber) {
    if (thirdNumber > secondNumber) {
        console.log("Increasing order")
    } else {
        console.log("Neither increasing or decreasing order")
    }
} else if (secondNumber < firstNumber) {
    if (thirdNumber < secondNumber) {
        console.log("Decreasing order")
    } else {
        console.log("Neither increasing or decreasing order")
    }
}

File: Edinburgh College/Python/2. Python Control Statements/activity2.py
# Control statements

# Get first number
first_number = int(input("Enter first number: "))

# Get second number
second_number = int(input("Enter second number: "))

# Get third number
third_number = int(input("Enter third number: "))

if third_number > second_number > first_number:
    print("Increasing order")
elif third_number < second_number < first_number:
    print("Decreasing order")
else:
    print("Neither increasing or decreasing order")